packages/kbn-optimizer/limits.yml

```diff
@@ -123,7 +123,7 @@ pageLoadAssetSize:
   security: 81771
   securitySolution: 66738
   securitySolutionEss: 16573
-  securitySolutionServerless: 45000
+  securitySolutionServerless: 62488
```

Comment:
OK :sweat_smile:

Reply:
Yeap.. We have both navigations implemented now, when we remove the old one this should go down